apps/abstraxion-dashboard/components/AbstraxionSignin/index.tsx

```diff
@@ -93,7 +93,7 @@ export const AbstraxionSignin = () => {
       return;
     }
     suggestAndConnect({
-      chainInfo: testnetChainInfo,
+      chainInfo: testnetChainInfo, // can't have chainInfo object as env var...
```

Comment:
Here

Reply:
Could we use a chain name in the env var? You could also JSON.stringify the config into a env var.

Reply:
Keplr requires the full chainInfo object when connecting. We can stringify the chainInfo, just wanted to see if there were other opinions with reasons I didn't see